src/Compatibility/Microsoft.DotNet.PackageValidation/BaselinePackageValidator.cs

```diff
@@ -56,7 +56,7 @@ public void Validate(Package package)
                         baselineCompileTimeAsset.Path,
                         package.PackagePath,
                         latestCompileTimeAsset.Path,
-                        Path.GetFileName(package.PackagePath),
+                        package.PackageId,
```

Comment:
The good thing about PackagePath though was that as you mention it has the version in it, so that makes it so that in the diff we can get more detailed messages when comparing against a baseline since we will get from which package version is each asset being selected from. I'm sure we can grab it from somewhere else, but for instance, I was using the fact that we had packagepath for the initial prototype of the warning suppressions.

Reply:
We are passing this for apicompat usage which requires just the assembly name. package.version contains the package version.